src/runtime-tools/win64/onefuzz.ps1

```diff
@@ -149,3 +149,14 @@ function Set-Restart {
   $config.restart = 'true'
   Write-OnefuzzConfig($config)
 }
+
+function Install-VCRedist {
+  log "Installing VC Redist"
+  $x64Release = 'https://aka.ms/vs/15/release/VC_redist.x64.exe'
+  $x86Release = 'https://aka.ms/vs/15/release/VC_redist.x86.exe'
+  Invoke-WebRequest -Uri $x64Release -OutFile "C:\onefuzz\vcredist_x64.exe"
```

Comment:
For better performance, the progress bar should be suppressed when using Windows PowerShell, see https://github.com/PowerShell/PowerShell/issues/2138#issuecomment-251165868

```suggestion
$ProgressPreference = 'SilentlyContinue'
Invoke-WebRequest -Uri $x64Release -OutFile "C:\onefuzz\vcredist_x64.exe"
```